controllers/controllers_test.go

```diff
@@ -88,6 +88,29 @@ var (
 	}
 )
 
+func newKubeconfigSecretForCluster(cluster *clusterv1.Cluster) *corev1.Secret {
+	return &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cluster.Name + "-kubeconfig",
+			Namespace: cluster.Namespace,
+		},
+		Data: map[string][]byte{
+			secret.KubeconfigDataName: []byte(`apiVersion: v1
```

Comment:
Nit: maybe we can move this into a var declaration outside the function so the indentation looks better like this:

```
var config = []byte(`apiVersion: v1
kind: Config
clusters:
- cluster:
    server: http://127.0.0.1:8080
  name: ` + cluster.Name + `
contexts:
- context:
    cluster: ` + cluster.Name + `
  name: ` + cluster.Name + `
current-context: ` + cluster.Name + `
`),
    },
  }
}
```

Reply:
Hard to move it outside as a var because it includes `cluster.Name` from the cluster argument. I've added new line at the start so indentation looks better but kept it in the function. Hope that's ok.